### DISCUSSION TITLE
Un authorized pool problem

### DISCUSSION THREAD

**Discussion by Zillos** [Chia-Network/chia-blockchain#11163] [Support]
I started for months plotting and farming with solo pool, then I realized I'm on a "Teal tiger pool", and I didn't accept that, it was against my will... can I delete it? if I delete it will cause any problems to my plots?

**Comment by DonMarmueta** [comment]
**
Are you sure it's in the pool? the plots that grow in the pool are plotted for the pool. If you haven't plotted with the pool address, it's all right.
However if you plotted to farm in a pool, you will have to re-plot.

Correct me if I'm wrong.**

**Comment by loppefaaret** [comment]
If you are talking about the "Pool" tab in the Chia client - these are plotNFT's that are controlling how your plots are being farmed. you can set your plotNFT to either farm solo, or farm using a public pool of your choice.
each plotNFT will get a name similar to "Teal Tiger" based on some random input.
it should show what its status are for each plotNFT
